server/webapp/WEB-INF/rails/webpack/views/pages/kitchen_sink.tsx

```diff
@@ -385,6 +387,15 @@ export class KitchenSink extends MithrilViewComponent<null> {
                      placeholder="password"
                      property={encryptedPasswordValue}
                      helpText={"Lorem ipsum dolor sit amet, consectetur adipiscing."}/>,
+      <RadioField label="Choose your favorite pizza topping"
+                  property={radioValue}
+                  required={true}
```

Comment:
Is it possible to have a radio button with no default selected (no value selected)?
Should we support such behavior?

Reply:
Technically, it's possible, since a form can be submitted without any selection. But from a usability point of view, it's unlikely that nothing would be selected. (since it's not trivial to un-select a radio-button). In some cases, we might want to start off with nothing selected, if we want the user to explicitly make a choice with no "default" given to them.

However, your comment made me realize that I haven't added a "checked" or "selected" property to select one of the options by default. I will add that to the `RadioData`

Reply:
Bound the `checked` property to the data model instead of a hardcoded boolean